crawl-ref/source/player-equip.cc

```diff
@@ -1277,7 +1246,7 @@ static void _equip_jewellery_effect(item_def &item, bool unmeld,
 
     if (!unmeld)
         mprf_nocap("%s", item.name(DESC_INVENTORY_EQUIP).c_str());
-    if (new_ident)
+    if (item.props["newly_identified"].get_bool())
         auto_assign_item_slot(item);
```

Comment:
I believe this should happen on pickup, so I think we can delete this (and the "newly_identified" prop in general). Please test that, though.

Reply:
All of this sounds good to me, I'll have to look at the item slot stuff since there were some rcfile things being called in there.

I probably won't get around to updating this until tomorrow, though.

Thanks PF~